build-system/npm-publish/write-package-files.js

```diff
@@ -164,6 +165,13 @@ async function writePackageJson() {
       preact: '^10.2.1',
       react: '^17.0.0',
     },
+    ...(useBentoCore
+      ? {
+          dependencies: {
+            '@bentoproject/core': `tbd`,
+          },
+        }
+      : {}),
```

Comment:
I don't love the way this looks, but I don't have a great suggestion aside from either collapsing some whitespace or moving it outside of the object literal into a temp const var

```suggestion
    ...(useBentoCore
      ? {dependencies: {'@bentoproject/core': 'tbd'}}
      : {}),
```

Reply:
linter keeps changing it to ` ...(useBentoCore ? {dependencies: {'@bentoproject/core': `tbd`}} : {}),` which is not horrible i guess, which I'm gonna change to. it'll change (soon) anyway, so hopefully not a huge deal